project/src/MainMenu/SteeringButton.gd

```diff
@@ -0,0 +1,16 @@
+extends TextureButton
+
+export var steering_string: String = "none"
+
+func handle_on_focus_entered() -> void:
+	set_scale(Vector2(1.3, 1.3))
```

Comment:
Skąd te wartości 1.3; 1.3, może warto wyciągnąć je do zmiennej?

Reply:
To jest w zasadzie niepotrzebne tak samo jak metoda niżej `handle_on_focus_exited`, zapomniałem tego usunąć

Reply:
To do wywalenia 🔨